archlinux/content.md

```diff
@@ -0,0 +1,32 @@
+# What is Arch Linux?
+
+Arch Linux, is a lightweight and flexible Linux® distribution that tries to Keep It Simple.
+
+Currently we have official packages optimized for the x86-64 architecture. We complement our official package sets with a community-operated package repository that grows in size and quality each and every day.
+
+Our strong community is diverse and helpful, and we pride ourselves on the range of skillsets and uses for Arch that stem from it. Please check out our forums and mailing lists to get your feet wet. Also glance through our wiki if you want to learn more about Arch.
+
+%%LOGO%%
+
+# About this image
+
+The base filesystem tarball for this image is auto-generated in Arch Linux infrastructure at the beginning of each month. Given the rolling-release nature of Arch Linux, images are tagged with the timestamp of the date they were generated. For example, `%%IMAGE`:20181107 was generated the Seventh of November of 2018. The latest tag will contain the latest build always.
```

Comment:
Once we're getting ready to merge, I think you'll want to update this example to whatever's the "latest" snapshot at the time so it's not referencing a tag that never existed. :smile:

Also, you'll want to make sure `%%IMAGE` is the full `%%IMAGE%%` and probably should include the tag inside the code markers, ala:

```markdown
For example, `%%IMAGE%%:20190905` was generated ...
```

Reply:
Noted. I'll keep this in mind.